config/routes/claims.rb

```diff
@@ -61,6 +61,11 @@
     end
 
     resources :schools, except: %i[destroy update] do
+      member do
+        put :remove_grant_conditions_acceptance
+        get :remove_grant_conditions_acceptance_check
+      end
```

Comment:
To make these seamless to the user, can we use the same path please? Also, Simon has nudged us to use dashes for word breaks.

```suggestion
      member do
        put :remove_grant_conditions_acceptance, path: "remove-grant-conditions-acceptance"
        get :remove_grant_conditions_acceptance_check, path: "remove-grant-conditions-acceptance"
      end
```